pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -197,6 +197,9 @@ type MonitoringSpec struct {
 
 	// ExternalMgrEndpoints points to an existing Ceph prometheus exporter endpoint
 	ExternalMgrEndpoints []v1.EndpointAddress `json:"externalMgrEndpoints,omitempty"`
+
+	// ExternalMgrServicesPort Prometheus exporter port
+	ExternalMgrServicesPort map[string]uint16 `json:"externalMgrServicesPort,omitempty"`
```

Comment:
If there are only one or two ports to expose, I'd rather create specific settings (like the suggestion above) instead of a general map

Reply:
I don't foresee more than 2 or 3 services eventually. Sounds fine to have dedicated ones I suppose...